docs/reference/analysis/synonyms.asciidoc

```diff
@@ -0,0 +1,261 @@
+[[analysis-synonyms]]
+== Synonyms
+
+Synonyms are words or phrases that have the same or similar meaning.
+They are an important aspect of search, as they can improve the search experience and the relevancy of search results.
```

Comment:
"improve relevancy" = > increase the scope

Usually synonyms reduce relevancy, but increase the recall (scope) of search results.

Reply:
Thanks Mayya! 💯